### PR TITLE
rg: add Turkish translation

### PR DESCRIPTION
**Pull request by eorus**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.tr/common/rg.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# rg
+
+> Ripgrep, yinelemeli satır-odaklı bir CLI arama aracıdır.
+> Grep'e daha hızlı bir alternatif olmayı hedefler.
```
Comment (kbdharun) [inline pages.tr/common/rg.md:4]:
```suggestion
> Grep'e daha hızlı bir alternatif olmayı hedefler.
```

File: pages.tr/common/rg.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+> Ripgrep, yinelemeli satır-odaklı bir CLI arama aracıdır.
+> Grep'e daha hızlı bir alternatif olmayı hedefler. 
+> Daha fazla bilgi: <https://github.com/BurntSushi/ripgrep>.
+
+- Normal bir ifade için geçerli dizini yinelemeli olarak arayın:
+
+`rg {{normal_ifade}}`
+
+- Geçerli dizinde, gizli dosyalar ve ".gitignore" da listelenen dosyalar dahil olmak üzere normal ifadeleri yinelemeli olarak arayın:
```
Comment (kbdharun) [inline pages.tr/common/rg.md:11]:
```suggestion
- Geçerli dizinde, gizli dosyalar ve ".gitignore" da listelenen dosyalar dahil olmak üzere normal ifadeleri yinelemeli olarak arayın:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

File: pages.tr/common/rg.md (hunk 1 above)
Comment (eorus) [reply pages.tr/common/rg.md:4]: You're welcome and thanks for fixing it.

File: pages.tr/common/rg.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+# rg
+
+> Ripgrep, yinelemeli satır-odaklı bir CLI arama aracıdır.
+> Grep'e daha hızlı bir alternatif olmayı hedefler.
+> Daha fazla bilgi: <https://github.com/BurntSushi/ripgrep>.
```
Comment (blueskyson) [inline pages.tr/common/rg.md:5]: Check out https://github.com/tldr-pages/tldr/blob/main/scripts/set-more-info-link.py#L41 .
```suggestion
> Daha fazla bilgi için: <https://github.com/BurntSushi/ripgrep>.
```

**Review by blueskyson — APPROVED** [review]

**Review by yutyo — APPROVED** [review]
LGTM

**Review by kbdharun — APPROVED** [review]

**Pull request merged**